faq/faq.md

```diff
@@ -0,0 +1,365 @@
+<!--
+SPDX-Copyright: 2019 Free Software Foundation Europe e.V.
+
+SPDX-License-Identifier: CC-BY-SA-4.0
+-->
+
+# General {#general}
+
+## I am short on time. Can you give me the quickest possible summary? {#quick-summary}
+
+We want to improve the way that people license their software projects. We
+propose three steps to achieve this:
+
+### 1. Choose and provide licenses {#step-1}
+
+Find the SPDX identifier of your license in the [SPDX License
+List](https://spdx.org/licenses/). Download the license text for your license
+from the
+[license-list-data](https://github.com/spdx/license-list-data/tree/master/text)
+repository and put it in the `LICENSES/` directory.
+
+### 2. Add copyright and license information to each file {#step-2}
+
+Then, for all files, edit the header to contain the following:
+
+```
+# SPDX-Copyright: [year] [copyright holder] <[email address]>
+#
+# SPDX-License-Identifier: [identifier]
+```
+
+### 3. Confirm REUSE compliance {#step-3}
+
+Use the [REUSE tool](https://github.com/fsfe/reuse-tool) to automate some of
+these steps, and to check whether you did everything correctly.
+
+## What is copyright? {#what-is-copyright}
+
+Copyright is a legal construct that grants someone exclusive rights over a
+creative work. The most important exclusive right is in the name: The right
+to produce copies. Only the copyright holder is allowed to give new copies of
+their work to people.
+
+Usually the author is the copyright holder, but often copyright is
+transferred to the author's employer. In most places, you do not need to do
+anything to gain copyright. As soon as you make a creative work, you (or your
+employer) instantly gain copyright over it.
+
+Creative Commons provides a better and lengthier answer in their [Frequently
+Asked
+Questions](https://creativecommons.org/faq/#what-is-copyright-and-why-does-it-matter).
+
+## What is a license? {#what-is-license}
+
+One problem with copyright as it pertains to software is that it makes software
+unshareable by default. A license changes that. A license defines the terms
+under which the copyright holder allows the recipient of the license to use the
+software.
+
+If the license allows the recipient to [use, study, share and
+improve](https://fsfe.org/freesoftware/basics/4freedoms.html) the software, then
+that software is Free Software.
+
+## Which license should I choose? {#which-license}
```

Comment:
Should we link to choosealicense.com?

Reply:
Are there other resources as well, e.g. explicitely recommended by the FSF?

Reply:
FSF has a very short recommendation list: https://www.gnu.org/licenses/license-recommendations.html

Both of these resources to have their own biases built in. I would suggest to point to both and perhaps mention that each chooser/resource emphasises a different value.

Reply:
Added in 1c895a0